docs/achtergronddocumentatie/artefacten/9018.md

```diff
@@ -1,46 +1,47 @@
+## 2. Authenticatie
 
 - Voor de afhandeling van het verzoek is het nodig de identiteit van de klant te weten. (De klant te 'kennen').
 - De medewerker voert de vereiste authenticatie procedure uit.
   - n.b. De standaard schrijft deze authenticatie procedure inhoudelijk niet voor. Hoe deze verloopt is afhankelijk van het noodzakelijke betrouwbaarheidsniveau.
-- De medewerker zoekt de klant in het relatiebeheersysteem ([F0138](./0138.md) en/of [F7927](./7927.md)).
+- Tijdens het authenticatie-proces wordt bekend of de klant bekend is als `partij`.
 
-### Variant A - Klant bekend
-- De klant kan gevonden worden in het relatiebeheersysteem.
-- De medewerker controleert of de geregistreerde contactgegevens van de klant nog actueel zijn.
-- Het kan zijn dat de klant in het kader van de productaanvraag afwijkende contactgegevens wil gebruiken. Deze worden dan in stap 3 vastgelegd als procesgegevens.
+## 3. Het `klantcontact`  wordt geregistreerd met behulp van [F2173 - registreer klantcontact](./2173)
 
-### Variant B - Klant niet bekend
-- De klant kan niet gevonden worden.
-- De medewerker vraagt aan de klant of deze zijn/haar contactgegevens wil doorgeven voor algemeen gebruik door de gemeente of alleen t.b.v. de afhandeling van het verzoek.
-  - Indien de klant zijn/haar contactgegevens voor algemeen gebruik wenst vast te leggen, wordt een nieuwe klant geregistreerd in het relatiebeheersysteem ([F775](./7554.md)).
-  - Wil de klant de contactgegevens niet herbruikbaar vastleggen - of - naast algemene gegevens afwijkende contactgegevens op nemen - dan worden deze in stap 3 vastgelegd.
-  
-## 3. Registratie zaak / interne taak en klantcontact
+Bij het vastleggen van het klantcontact worden de volgende punten in acht genomen:
 
-### Variant A - Registratie zaak
-- De medewerker maakt direct een `zaak` aan voor de afhandeling van de productaanvraag.
-- Indien de klant heeft opgegeven dat de contactgegevens uitsluitend gebruikt mogen worden in het kader van deze productaanvraag - of - afwijkende contactgegevens heeft opgegeven, dan worden deze contactgegevens als onderdeel van deze `zaak` geregistreerd.
+- Er wordt een onderwerp ingevuld in het *onderwerp* van het klantcontact.
+- Eventueel wordt er een samenvatting of beschrijving van de productaanvraag opgenomen in de *inhoud* 
+- Het vastleggen van klant-gegevens is beschreven in de paragraaf  [Vastleggen van de betrokkeneBijKlantcontact](./2173.md#vastleggen-betrokkenebijklantcontact)
+   
+## 4. Registreer Bijlage (productaanvraag).
+
+- De gegevens over de productaanvraag wordt uitgevraagd bij de klant en vastgelegd middels de daartoe bestemde applicatie.
+- Nadat de productaanvraag is vastgelegd (al dan niet compleet) wordt er een `bijlage` vastgelegd met de functie [F1975 - Registreer bijlage](./1975.md)
+- in de bijlage wordt de verwijzing naar het digitale object van de productaanvraag vastgelegd en de bijlage wordt aan hetr klantcontact gekoppeld. 
```

Comment:
Ik weet niet of dit naar aanleiding van een telefoongesprek met het KCC een realistisch scenario is - ik vermoed dat de meeste KCC's geen 'complexe' productaanvragen namens klanten kunnen registreren. Het kan ook een hypothetische casus zijn natuurlijk - dan is het misschien goed nader te duiden waar die gegevens in de bijlage vandaan komen en in welke vorm ze worden geregistreerd.

Reply:
Ik heb de casus beschrijving aangepast